### DISCUSSION TITLE
Add dev tooling support

### DISCUSSION THREAD

**Discussion by Guldem** [lejard-h/chopper#621] [Ideas]
Although the dart dev tool already have support for http call inspections. It might still be interesting to have some chopper specific dev tool support. 

Also this could maybe be kickstarted by using something like [Alice](https://github.com/jhomlala/alice). The drift package also enable dev tool support with a third party package.

**Comment by techouse** [comment]
This would be very cool. Currently one has to look at the Flutter logs for CURL outputs, which kinda works but is also kinda annoying.

**Comment by Guldem** [comment]
I was also thinking that if possible it would be cool to see the changes made by interceptors and maybe even converters. To give developers more tools to debug stuff. 

|Interceptor|Request before|Request after|
|-----------|---------------| -------------|
|HeaderInterceptor| Request A `{'myHeader':'123}`| Request A `{'myHeader':'123, 'AddedHeader': 'Interesting'}`|

**Comment by Guldem** [comment]
I sort off lstarted working on dev_tools support. Its not much atm but hopefully I have something to share soon.
